cookbook/deployment/cluster/config_flyte_deploy.py

```diff
@@ -1,8 +1,187 @@
 """
+#################################
 Configuring Your Flyte Deployment
---------------------------------------------
+#################################
 
-.. NOTE::
+As the complexity of your user base grows, you may find yourself tweaking resource assignments based on specific projects, domains and workflows. This document walks through how and in what ways you can configure your Flyte deployment.
+
+
+***************************
+Configurable Resource Types
+***************************
+
+Flyte allows these custom settings along the following combination of dimensions:
+
+- domain
+- project and domain
+- project, domain, and name (must be either a workflow name or a launch plan name)
+
+Please see the :ref:`divedeep-projects` document for more information on projects and domains. Along these dimensions, the following settings are configurable. Note that not all three of the combinations above are valid for each of these settings.
```

Comment:
Please confirm this document is up to date and can be accessed (there is currently no link)